sample/pubspec.yaml

```diff
@@ -31,11 +31,14 @@ dependencies:
     sdk: flutter
   blip_ds:
     path: ../
+  just_audio: ^0.9.27
```

Comment:
Seems that the just_audio, rxdart and flutter_spinkit packages aren't needed in Sample's pubspec.yaml.

Reply:
i removed unused dependencies